### ISSUE TITLE
Can I disable diagnostics for cjk? 

### ISSUE THREAD

**Issue by Jaehaks** [elijah-potter/harper#109]
Thank you for your efforts.

Can I set diagnostics range of harper_ls?
All cjk characters are detected as error.

**Comment by elijah-potter** [comment]
Absolutely. This should actually be a pretty simple fix. No promises, but I might get a release out today with it.

**Comment by Jaehaks** [comment]
@elijah-potter
I hope so, Thanks :)

**Comment by elijah-potter** [comment]
I've published a release that should ignore CJK characters. Eventually, I'll want to support grammar checking for Chinese, Japanese and Korean, but for now it'll have to do. 

Once Mason has pulled the newest version, would you mind running it on a couple of your documents to make sure everything checks out? I _believe_ I've covered the relevant character sets, but would love a smoke-test.

If you don't see any issues, go ahead and close this issue.
Thank you so much for letting me know this was bothering you!

**Comment by Jaehaks** [comment]
@elijah-potter 
Thank you for your fast response,

Unfortunately, Harper-ls detects non-english word as error(hint). 
After update of mason and harper-ls, it doesn't look like it's working well

**Comment by elijah-potter** [comment]
Thanks for checking! I hoped that we could simply have Harper ignore certain Unicode blocks. I've got another solution in the words. In the meantime, could you provide me a few examples of the prevailing issue?

**Comment by Jaehaks** [comment]
@elijah-potter 

Any file which use korean will be examples.
I'll attach a simple example as this link : [test.md](https://github.com/user-attachments/files/16638708/test.md)

I have two windows PCs, 
Nvim in one PC makes both lsp diagnostics and underline, but nvim in the other PC makes only lsp diagnostics although I use the same configuration dotfile of neovim. 
I don't know why they have difference results.

**Comment by elijah-potter** [comment]
I've expanded the CJK classifier accordingly. I hope it's OK that I added your example document as a test case. Let me know if it's not.

In regard to the other problem (with the underlining), I don't think that is a Harper issue. Try making sure both computers have the same version of Neovim and terminal emulator.

Once Mason picks up the new release, give it another spin and give me an update if it works.
